app/src/main/java/ch/heigvd/pro/b04/android/poll/PollViewModel.kt

```diff
@@ -0,0 +1,19 @@
+package ch.heigvd.pro.b04.android.poll
+
+import android.app.Application
+import androidx.lifecycle.MutableLiveData
+import ch.heigvd.pro.b04.android.datamodel.Question
+import ch.heigvd.pro.b04.android.network.RequestsViewModel
+
+class PollViewModel(application: Application,
+                    idModerator : Int,
+                    idPoll : Int,
+                    token : String
+) : RequestsViewModel(application, idModerator, idPoll, token) {
+    val questionToView : MutableLiveData<Question> = MutableLiveData()
```

Comment:
Not necessary, but we may be able to use a `MutableStateFlow`.

Reply:
Will do this in another PR (probably)